### DISCUSSION TITLE
Bookmarks can't be synced

### DISCUSSION THREAD

**Discussion by gloryandel** [floccusaddon/floccus#1119] [Q&A] [answered]
Why does bookmark lock (.block) file occasionally appear, which prevents normal synchronization?

**Comment by marcelklehr** [comment, marked as answer]
The lock file is used to avoid data loss when two browsers of the same user sync at the same time.

**Reply by gloryandel** [reply]
But because of this, I had to manually delete the files to achieve restart synchronization, because neither of them could actively delete them, but neither of them would give up synchronization, always showing as synchronizing, causing a deadlock! 😂😂

**Reply by marcelklehr** [reply]
There is a mechanism that automatically removes the lock file after waiting some time.
